sql/core/src/test/scala/org/apache/spark/sql/JsonFunctionsSuite.scala

```diff
@@ -39,6 +39,13 @@ class JsonFunctionsSuite extends QueryTest with SharedSparkSession {
       Row("alice", "5"))
   }
 
+  test("function get_json_object - support single quotes") {
+    val df: DataFrame = Seq(("""{'name': 'fang', 'age': 5}""")).toDF("a")
```

Comment:
nit: `val df = Seq(("""{'name': 'fang', 'age': 5}""")).toDF("a")`

Reply:
In this class of JsonFunctionsSuite , other tests are in the following format,
`val df: DataFrame = tuples.toDF("key", "jstring")`
I want to keep the same format as other tests.